### PR TITLE
Add sample to get access token by oauth

### PR DESCRIPTION
**Pull request by lamrongol**
Modify: add redirectUri

### CODE REVIEW

**Review by andregasser — CHANGES_REQUESTED** [review]
Hello @lamrongol, sorry for the long wait time. I just had another look at this PR and I think it would be best to have your code in the repo as a working example in the sample-* modules instead of just listing it in USAGE.md. Could you eventually provide a new piece of sample code to the sample-java module (separate file?). Reason: We would like to remove the USAGE.md sooner or later. If you don't have the time to so it, I could do it. Just let me know. Thank you! 😊

**Comment by lamrongol** [comment]
OK, but what should the file name, etc. be? If I should simply cut and paste the code somewhere else, it would be easy and I want to leave it to you.

**Comment by andregasser** [comment]
You could name it 'OAuthGetAccessToken.java'. Thank you very much!

**Comment by lamrongol** [comment]
I modified. Kotlin code is converted by IntelliJ IDEA and I confirmed it works.

File: sample-kotlin/src/main/kotlin/social/bigbone/sample/OAuthGetAccessToken.kt
Comment (andregasser) [inline sample-kotlin/src/main/kotlin/social/bigbone/sample/OAuthGetAccessToken.kt:6]: Please replace star import with java.util.Scanner

**Review by andregasser — CHANGES_REQUESTED** [review]
Thanks for providing the sample code! I have just added a minor comment to your PR. After fixing this, I will happily merge it into the repo.

**Comment by lamrongol** [comment]
"Unresolved conversations" is still displayed for some reason, but this has been fixed.

**Review by andregasser — APPROVED** [review]

**Comment by andregasser** [comment]
Thanks @lamrongol for fixing this so quick!

**Pull request merged**
